go-controller/pkg/informer/informer_test.go

```diff
@@ -91,7 +92,7 @@ var _ = Describe("Informer Event Handler Tests", func() {
 		defer close(stopChan)
 
 		f.Start(stopChan)
-		go e.Run(1, stopChan)
+		go e.Run(1, stopChan, &sync.WaitGroup{})
```

Comment:
You should define `wg` here and ensure we `wg.Wait()` in the test.

Reply:
Done, and added BeforeEach/AfterEach to handle setup/cancellation of the channel and waitgorup